src/test/java/org/apache/commons/lang3/exception/ContextedExceptionTest.java

```diff
@@ -48,6 +48,11 @@ public void testContextedException() {
         assertTrue(StringUtils.isEmpty(message));
     }
 
+    @Test
+    public void testNullException() {
+        assertTrue(StringUtils.isEmpty(ExceptionUtils.getStackTrace(null)));
```

Comment:
I think the test can be simplified by not using StringUtils.

Reply:
HI @garydgregory 
SomeThins like --> `assertNotNull(ExceptionUtils.getStackTrace(null));` ??
TY

Reply:
> 
> 
> HI @garydgregory
> SomeThins like --> `assertNotNull(ExceptionUtils.getStackTrace(null));` ??
> TY

Hello @arturobernalg 
No, assert what you expect to happen on null input. Your suggestion above would accept a return value of "foo". Is that OK? I think not.

Reply:
HI @garydgregory 
ok i think i understand what you mean.
TY